### ISSUE TITLE
SFT loss vs LM loss

### ISSUE THREAD

**Issue by Jiaxin-Wen** [TIGER-AI-Lab/MAmmoTH2#7]
Figure 6 compares the effectiveness of two training losses: LM loss and SFT loss.
However, I do not find any formal definition of these two losses.
By SFT loss, do you mean masking question tokens while calculating the loss?

**Comment by wenhuchen** [comment]
Exactly. SFT loss means the "prefix LM loss" where the question is not given any loss.
